doc/api/v0.yaml

```diff
@@ -56,6 +56,45 @@ paths:
                 $ref: "#/components/schemas/Fund"
         "404":
           description: The requested fund was not found
+    put:
+      operationId: putFund
+      summary: Update or create fund
+      tags: [fund]
+      description: |
+        Retrieves information on the identified treasury fund campaign.
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+      responses:
+        "200":
+          description: Valid response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Fund"
+        "422":
+          description: The input is malformed.
+
+  /api/v0/funds/:
+    get:
+      operationId: getFunds
+      summary: Get list of the funds
+      tags: [fund]
+      description: |
+        Get list of all the funds in the db. Ordered in time, which means the first one is the current one.
```

Comment:
maybe it's enough to return all the funds ids? I'm not sure there are a lot of cases where you want to fetch all of them

Reply:
Well, it is mostly to save a request, since this would be queried all the time.

And I was expecting that we just add the two that we care about for now, we can add pagination arguments in the future (like `start=current&limit=2`) if necessary.

Another option would be to just return the current and the next in `v0/fund`, since that endpoint is a bit strange anyway, and is the one that the app currently uses, and keep this only for ids.

Reply:
My understanding is that we want to keep old entries as well, how do we handle those?

>which means the first one is the current one.

like this would not be true anymore I think

Reply:
If we add old funds, yes, but are we going to do that? 

Currently you can just have the current one. And even here I only removed the restriction on having one, but I'm still using the first one as the current (and `v0/funds` does the same).

But I can remove that part of the comment for it to be more generic.

Reply:
Depends on requirements, when I read `continuous` I interpreted it as not restarting the instance each fund but rather add new stuff, which would mean that we return old stuff as well. But if that's not needed we can just restart it without any past data